src/scancode/cli.py

```diff
@@ -785,4 +776,4 @@ def save_results(scanners, only_findings, files_count, results, format, options,
                     raise e
         return
 
-    write_formatted_output(scanners, files_count, version, notice, results, format, options, input, output_file, echo_stderr)
+    write_formatted_output(files_count, version, notice, results, format, options, input, output_file, echo_stderr, scan_output_plugins)
```

Comment:
To the contrary... I think the whole purpose of a PluginManager is to avoid the type of `plugin_formats` global you have been using